### PR TITLE
fix[backend]: fix claimsfeed2

### PR DESCRIPTION
**Pull request by Kholoudxs55kh**

### CODE REVIEW

File: prisma/migrations/20240802000848_image_migration_aug_1_2024/migration.sql [hunk 1]
```diff
@@ -0,0 +1,72 @@
+-- CreateEnum
```
Comment (gvelez17) [inline prisma/migrations/20240802000848_image_migration_aug_1_2024/migration.sql:1]: @Kholoudxs55kh PLease do NOT NOT NOT EVER include migrations in important hotfix :-)

migrations can potentially corrupt data, they cannot be rolled back, they should ALWAYS be in SEPARATE PRS

THANK YOU

Comment (Kholoudxs55kh) [reply prisma/migrations/20240802000848_image_migration_aug_1_2024/migration.sql]: @gvelez17 Removed Them

**Review by AhmedAbdelmenam — APPROVED** [review]
